packages/spacecat-shared-rum-api-client/test/index.test.js

```diff
@@ -31,6 +33,9 @@ describe('RUMAPIClient', () => {
   });
 
   it('runs the query', async () => {
+    nock('https://rum.fastly-aem.page')
```

Comment:
unrelated, but encountered a test failure with the test accessing the live API and getting a `401`